axelar-chains-config/info/testnet.json

```diff
@@ -1,5 +1,226 @@
 {
   "chains": {
+    "test-avalanche": {
```

Comment:
move it to the bottom since it's a test instance?

Reply:
will remove it soon no ?